### ISSUE TITLE
Translate page resources to French.

### ISSUE THREAD

**Issue by MarconiGRF** [MarconiGRF/YTGameBarOverlay#4]
## To contribute with the translation for your language follow these steps:

1. Find the correct **culture code** for your language [in this list](https://docs.microsoft.com/en-us/bingmaps/rest-services/common-parameters-and-types/supported-culture-codes) and remember it.
2. On the `Resources` folder it will have each page resource files, create a new .xml with the filename of your **culture code**. For example, if your **culture code** is `pt-BR` create `pt-BR.xml` in every page folder.
3. Replace the strings of the `en-US.xml` version on your version with the correct string translations.
4. Commit the changes and open a new pull request.

**Comment by Jupiteur** [comment]
Hi, I'm currently working on a French translation for YTGBO (https://github.com/Jupiteur/YTGameBarOverlay), is there any other configuration required?
I'm thinking of the language selection settings (I added a `<FrenchFR>` field in all xml files of the SettingsRessouirces folder). But I don't know if the code will fetch it on its own.

**Comment by MarconiGRF** [comment]
Hi @Jupiteur! 

First of all thanks for the work on the translation. 

TL;DR - You only need to worry about the .xml files translations, I can do the remaining tasks about the language being available on options.

Secondly, the code will get the options but just adding the `FrenchFR` field would not be enough, you'll need to add the french option [to the constants options](https://github.com/MarconiGRF/YTGameBarOverlay/blob/e46bd3b42dd950d0ed6c1aa9787fa9ad41a80681/Utilities/Constants.cs#L129C1-L139C1). Although this is needed to make it work you may not need to do that, before the Pull Request gets merged to `master` I always make the necessary checks to ensure everything is working as expected (including adding the missing options).

**Comment by MarconiGRF** [comment]
Thanks for the work @Jupiteur! Although it was merged I won't plan updating the app due to issues, for more info please refer to the deprecation notice on the README and feel free to publish and redistribute the app with this code.
